src/Telnyx.net/Services/Wireless/WDRReports/ListWDRReportsOptions.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace Telnyx.net.Services.Wireless.WDRReports
+{
+    public class ListWDRReportsOptions : ListOptions
```

Comment:
Remove file since its just ListOptions. Unless the idea is that if the api changes to support more parameters?

Reply:
done.